news/_posts/2018-03-19-vmtk-1.4.0-released.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: news
+post-type: announcement
+title: "VMTK 1.4.0 Released"
+---
+
+We are pleased to announce the latest version of VMTK is ready for general use! Please see the documentation below for details.
+
+## Python 3 Support
+VMTK now supports the following python versions for each platform:
+
+- Linux (Many Linux Compatible): 2.7, 3.5, 3.6+
+- MacOSX (Minimum MacOS 10.9): 2.7, 3.5, 3.6+
+- Windows 10: 3.5, 3.6+
+
+
+----------
+
+## Packaging and Distribution via Anaconda
+VMTK has officially embraced the Anaconda Python packaging and distribution system for dependency and code management. Official releases are posted to the [VMTK Anaconda Channel](https://anaconda.org/vmtk/repo). Our [Installation Instruction](http://www.vmtk.org/download/) have been updated on the VMTK webpage to reflect the change. 
+
+Executable installers are also available at the [VMTK Downloads Page](http://www.vmtk.org/download/).
+
+In the future, we plan to also post releases to the [conda-forge](https://conda-forge.org/) Anaconda channel. We are waiting for conda-forge to support VTK on MacOSX before making this transition. 
+
+Please note: we will no longer be releasing new versions of VMTK on PyPi
+
+
+----------
+
+## Native Conversion of VMTK Data Structures to Numpy Arrays
+We now allow you to use all of the functionality that VMTK offers in filtering images, creating surfaces,  centerlines, or volumetric meshes, while seemlessly being able to employ any external processing available in the scientific python ecosystem. With the following new methods, you can losslesly translate data (generated in either VMTK or in Python proper) back and forth between the VTK data formats and Numpy Arrays:
+
+- vmtkImageToNumpy
+- vmtkNumpyToImage
+- vmtkCenterlinesToNumpy
+- vmtkNumpyToCenterlines
+- vmtkSurfaceToNumpy
+- vmtkNumpyToSurface
+- vmtkMeshToNumpy
+- vmtkNumpyToMesh
+
+A [tutorial has been provided](http://www.vmtk.org/tutorials/WorkingWithNumpyArrays.html) on the VMTK website which walks you through the process of interacting with the translation routines
+
+
+----------
+
+## Store Data to Disk as HDF5 or Pickled Python Files
+With the VMTK-Numpy conversion utilities, we now allow VMTK data structures to be saved (and read) to disk as HDF5 or Python Pickle objects. These methods are available in the following methods:
+
+- vmtkNumpyWriter
+- vmtkNumpyReader
+
+
+----------
+
+## Pype Scripts Include Sensible Default Values
+
+It is now significantly easier to break up pypes run on the command line. Instead of relying on output members to specify input array names for scripts down the pype, we now include sensible defaults to the input array names. It is now possible to run one set of commands which generate a surface/centerline from an image, write those files to disk, and run another set of commands which perform further calculations or visualizations with no need to respecify array names.
```

Comment:
Probably a very quick example would be useful here (like something involving `radiusarray`), something like
```
vmtkbranchextractor -ifile centerlines.vtp -ofile centerlines_split.vtp
```
instead of
```
vmtkbranchextractor -ifile centerlines.vtp -radiusarray MaximumInscribedSphereRadius -ofile centerlines_split.vtp
```